scripts/g.extension/g.extension.py

```diff
@@ -2538,20 +2539,16 @@ def get_addons_paths(gg_addons_base_dir):
     in the $GRASS_ADDON_BASE dir. The file serves as a list of all addons,
```

Comment:
Is there supposed to be addons_paths.json instead of extensions_paths.json?

Reply:
Thanks for catching this. It looks like I'm confused by these two concepts extensions vs. addons.

Reply:
> Thanks for catching this. It looks like I'm confused by these two concepts extensions vs. addons.

There was an open issue/discussion regarding the use of these terms. AFAIU, they are used indistinctly, i.e., addon=extension. We should probably choose one and stick to it for consistency. The module to install addons is called g.extension but the extensions repo is called addons 🤯🤪